index.html

```diff
@@ -466,347 +466,365 @@ <h3>Ethereum EIP712 Signature 2021</h3>
-        "id":"https://example.org/person/1234","issuanceDate":"2010-01-01T19:23:24Z","issuer":"did
-        :example:aaaabbbb","proof":{"created":"2021-07-09T19:47:41Z","proofPurpose":"assertionMeth
-        od","type":"EthereumEip712Signature2021","verificationMethod":"did:example:aaaabbbb#issuer
-        Key-1"},"type":["VerifiableCredential"]},"primaryType":"VerifiableCredential","types":{"Cr
-        edentialSchema":[{"name":"id","type":"string"},{"name":"type","type":"string"}],"Credentia
-        lSubject":[{"name":"type","type":"string"},{"name":"id","type":"string"},{"name":"name","t
-        ype":"string"},{"name":"child","type":"Person"}],"EIP712Domain":[{"name":"name","type":"st
-        ring"},{"name":"version","type":"string"},{"name":"chainId","type":"uint256"},{"name":"sal
-        t","type":"bytes32"}],"Person":[{"name":"type","type":"string"},{"name":"name","type":"str
-        ing"}],"Proof":[{"name":"verificationMethod","type":"string"},{"name":"created","type":"st
-        ring"},{"name":"proofPurpose","type":"string"},{"name":"type","type":"string"}],"Verifiabl
-        eCredential":[{"name":"@context","type":"string[]"},{"name":"type","type":"string[]"},{"na
-        me":"id","type":"string"},{"name":"issuer","type":"string"},{"name":"issuanceDate","type":
-        "string"},{"name":"credentialSubject","type":"CredentialSubject"},{"name":"credentialSchem
-        a","type":"CredentialSchema"},{"name":"proof","type":"Proof"}]}}
-      </pre>
+      <section>
+        <h3>Basic Document - Types Generation - Embedded Types</h3>
 
-      <p>The following is the resulting <code>proof</code> object:</p>
-      <pre class="example">
-        {
-          "type": "EthereumEip712Signature2021",
-          "created": "2021-07-09T19:47:41Z",
-          "proofPurpose": "assertionMethod",
-          "proofValue": "0x5fb8f18f21f54c2df8a2720d0afcee7dbbb18e4b7a22ce6e8183633d63b076d329122584db769cd78b6cd5a7094ede5ceaa43317907539187f1f0d8875f99e051b",
-          "verificationMethod": "did:example:aaaabbbb#issuerKey-1",
-          "eip712Domain": {
-            "domain": {
-              "chainId": 4,
-              "name": "https://example.com",
-              "salt": "0x000000000000000000000000000000000000000000000000aaaabbbbccccdddd",
-              "version": "2"
-            },
-            "messageSchema": {
-              "CredentialSchema": [
-                {
-                  "name": "id",
-                  "type": "string"
-                },
-                {
-                  "name": "type",
-                  "type": "string"
-                }
-              ],
-              "CredentialSubject": [
-                {
-                  "name": "type",
-                  "type": "string"
-                },
+        <p>
+          With the following <code>inputOptions</code> provided to the signature suite along with the <code>testBasicDocument</code> input document:
```

Comment:
Could `inputOptions` and the meaning of the particular options be defined - either for the suite itself, or just for the test vectors?